plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -330,6 +330,11 @@ export class PersonState {
             return
         }
 
+        // The first distinct_id that is used in identify or alias (not the alias_id nor anon_id)
+        // cannot be used as alias_id nor anon_id in the future.
```

Comment:
Nit: Let's use correct terminology - there's no such thing as `alias_id` or `anon_id`. Someone reading this comment will start grepping for these and not find anything, resulting in confusion.